public/home.json

```diff
@@ -78,5 +78,27 @@
         "type": "desktop"
       }
     ]
+  },
+
+  "fullBannerSection": {
+    "title": "Become a Mentor",
+    "description": "Ready to empower and be empowered in tech? Become a mentor! Expand your network, give back, share expertise, and discover new perspectives.",
+    "link": {
+      "label": "Join as a mentor",
+      "uri": "https://docs.google.com/forms/d/e/1FAIpQLSdtf7-upMp1m5kJ4MSpexS-UwGJHhACEW-yPoEQoROHi4kVcg/viewform",
```

Comment:
This will be internal link, /mentorshiop/mentor-registration. But considering is mocking is no problem.